src/SdfGenerator.cc

```diff
@@ -422,6 +429,41 @@ namespace sdf_generator
     return true;
   }
 
+  void updateModelElementWithNestedInclude(sdf::ElementPtr &_elem,
+                                           const bool _saveFuelVersion)
+  {
+    sdf::ElementPtr e = _elem->GetFirstElement(), nextE = nullptr;
+    while (e != nullptr)
+    {
+      nextE = e->GetNextElement();
+
+      if (e->GetIncludeElement() != nullptr)
+      {
+        if (_saveFuelVersion && e->FilePath().find("fuel") != std::string::npos)
+        {
+          // find fuel model version from file path
+          std::string version = common::parentPath(e->FilePath());
+          version = common::basename(version);
+
+          sdf::ElementPtr uriElem = e->GetIncludeElement()->GetElement("uri");
+          std::string uri = uriElem->GetValue()->GetAsString();
+          uri = common::joinPaths(uri, version);
```

Comment:
`joinPaths` might use `\` delimiter on windows, but since this is a Fuel uri (https://...), we always want `/`.